src/tasks/helpers/base.ts

```diff
@@ -94,21 +94,38 @@ export abstract class TaskHelper {
 
 		if (!definition.outputDirectory) {
 			const defaultOutput = path.join(definition.projectDir, 'dist');
-			const useDefaultOutputChoice = await yesNoQuestion({ placeHolder: `Use default output directory ${defaultOutput}?` }, false, [ 'Yes', 'Choose custom output directory' ]);
-			if (useDefaultOutputChoice) {
-				definition.outputDirectory = defaultOutput;
-			} else {
+
+			const options = [ `Defualt: ${defaultOutput}`, 'Browse' ];
+
+			if ((definition as AppPackageTaskTitaniumBuildBase).target === 'dist-appstore') {
+				options.push('Output Into Xcode');
+			}
+
+			const selected = await quickPick(options, {
+				canPickMany: false,
+				placeHolder: 'Choose output location'
+			}, {
+				forceShow: true
+			});
+
+			if (selected === 'Output Into Xcode') {
+				return;
```

Comment:
Returning early here might make things more awkward if we need to expand upon this function in future. Maybe do nothing if `Output Into Xcode` is selected in the quick pick, assign `definition.outputDirectory` if `Browse` or `Default` was selected. Then on line 111 call `builder.addQuotedOption('--output-dir', definition.outputDirectory);` only if `definition.outputDirectory` exists